ar_SA.json

```diff
@@ -1616,6 +1621,11 @@
 	"property.label.tls.maxVersion": "TLS Maximum Version",
 	"property.label.tls.minVersion": "TLS Minimum Version",
 	"property.label.tls.version": "إصدارات TLS المدعومة",
+	"property.label.tlsClientCACertId":"الشهادة المستخدمة للتحقق من العميل",
+	"property.label.tlsClientCACertId.invalid":"يرجى تحديد شهادة CA التي سيتم استخدامها للتحقق من العميل.",
+	"property.label.tlsClientCACertId.tips":"حدد شهادات CA الموثوقة التي يجب استخدامها للتحقق من شهادات العميل.",
+	"property.label.tlsVerifyClient":"التحقق من شهادة العميل",
+	"property.label.tlsVerifyClient.tips":"Enable or disable verification of client certificates. Clients that fail the verification will be rejected. \"On\" means verification is enabled.\"Off\" means verification is disabled. \"Optional\" means verification will be performed only when client presents certificates. \"Optional, no CA\" bears the same meaning as \"Optional\" except that the server won't verify if certificates presented by client are signed by trusted CA.",
```

Comment:
Not included the Arabic translation due to formatting issues involving double quotes. Intend to keep it as it is for now.

Reply:
OK. Since we don't officially support that language now, it's not so important.